test/e2e/framework/test_context.go

```diff
@@ -37,6 +37,9 @@ import (
 
 const defaultHost = "http://127.0.0.1:8080"
 
+// ListImages will list off all images that are used then quit
+var ListImages bool
```

Comment:
This seems weirdly global vs adding to the context object.

Reply:
I moved it to the test context object. My initial thought was this isn't really part of a test, so it didn't belong there.